tests/utilities/encodedImageStorage/uploadEncodedImage.spec.ts

```diff
@@ -180,7 +182,7 @@ describe("src -> utilities -> encodedImageStorage -> uploadEncodedImage", () =>
       fs.unlink(path.join(__dirname, "../../../".concat(fileName)), (err) => {
         if (err) throw err;
       });
-    } catch (error: any) {
+    } catch (error: unknown) {
```

Comment:
It should work fine with any, right?

Reply:
"any" will fail the linting, so that's why I had to use unknown